File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // ask for an height between the 1 and 23
    int height;
    do
    {
        printf("Height: ");
        height = get_int();
    }
    while(height > 23 || height <0);


    // print the whole piramid
    for(int i=1; i<=height; i++)
    {

        // print the spaces before the #'s  
        for(int k=0; k<(height-i); k++)
            {
                printf(" ");
            }

        // print first #'s
        for(int j=i; j>0; j--)
            {
                printf("#");
            }

        // print the two spaces        
            printf("  ");    

        // print the last #'s            
        for(int z=i; z>0; z--)
            {
                printf("#");
            }            

        // print a last return            
        printf("\n");
    }
}

File: mario.py
import cs50

def main():
    # ask for an height between the 1 and 23
    while True:
        print("Height: ")
        height = cs50.get_int()
        if height > 0 and height < 23:
            break

    # print the whole piramid
    for i in range(height):

        # print the spaces before the #'s  
        for k in range(height-i+1):
            print(" ", end="")

        # print first #'s
        for j in range(i+1):
            print("#", end="")
            
        # print the two spaces        
        print("  ", end="")

        # print the last #'s            
        for z in range(i+1):
            print("#", end="")

        # print a last return            
        print()

if __name__ == "__main__":
    main()